pkg/cmd/server/origin/aggregator.go

```diff
@@ -42,6 +42,13 @@ import (
 	informers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion"
 )
 
+func init() {
+	// install our types into the scheme so that "normal" RESTOptionsGetters can work for us.
+	// done in init() so we don't mutate the scheme after it is being used by clients in other goroutines.
+	// TODO: move to a scheme-per-process like upstream and register into that instead.
+	install.Install(kapi.GroupFactoryRegistry, kapi.Registry, kapi.Scheme)
```

Comment:
this wasn't for the integration test, it was the fix. I can put it in an isolated install package and import it just for our server wiring, but it's safer to do this in an init() in 3.6